frontend/public/components/utils/managed-by.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { Link } from 'react-router-dom';
+
+import { ResourceIcon } from './resource-icon';
+import {
+  groupVersionFor,
+  K8sResourceCommon,
+  K8sResourceKind,
+  referenceForGroupVersionKind,
+  referenceForModel,
+  referenceForOwnerRef,
+  OwnerReference,
+} from '../../module/k8s';
+import { ClusterServiceVersionModel } from '@console/operator-lifecycle-manager/src/models';
+import { ClusterServiceVersionKind } from '@console/operator-lifecycle-manager';
+import { k8sList } from '../../module/k8s/resource';
+
+const isOwnedByOperator = (csv: K8sResourceKind, owner: OwnerReference) => {
+  const { version } = groupVersionFor(owner.apiVersion);
+  return csv.spec.customresourcedefinitions.owned.some(
+    (api) => api.kind === owner.kind && version === api.version,
+  );
```

Comment:
You don't need to check version. You need to check API group instead. (Version doesn't matter here. The resources are the same resources regardless of API version as long as they're in the same group.)

```suggestion
  const { group } = groupVersionFor(owner.apiVersion);
  return csv.spec?.customresourcedefinitions?.owned?.some((owned) => {
      const ownedGroup = owned.name.substring(owned.indexOf('.') + 1);
      return owned.kind === owner.kind && group === ownedGroup;
    }
  );
```

Reply:
This doesn't run, but I came up with something that seems to be in line with what you meant?